### PR TITLE
proxy support for Junyper

### PR DESCRIPTION
**Pull request by fleXible**
Just finished adding proxy support to the glue script. My development HASS is running in a docker container inside a VM on my NAS. In this case networking tends to get suprisingly complex real quick so I had to install and run Junyper inside the HASS container.

Now an installed SSH server and a SOCKS5 tunnel using DynamicForward does the trick. Hopefully I didn't shuffle around too much code ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +55,8 @@ of the files (`kernel.json`, `hass_pyscript_kernel.py`, `pyscript.conf`, and the
     - Since you've added a HASS access token to this file, you should make sure you are
       comfortable with file permissions - anyone who can read this file could use the
       access token to use the HASS UI without being an authenticated user.
+    - `hass_proxy` if you use SSH to access your HASS instance, you can open a SOCKS5 tunnel 
+      keep your Jupyter local.
```
Comment (craigbarratt) [inline README.md:59]: This should explain what the setting actually is.  Also, is this missing "to" before "keep"?

File: pyscript.conf [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-#
-# Do not use quotes
-#
-hass_token = REPLACE_WITH_THE_LONG_TERM_ACCESS_KEY_FROM_HASS
+
+## Set hass_host to the host name or IP address where your HASS is running.
+## This should be a host name or IP address you can ping from the Jupyter
+## client machine. e.g., 192.168.1.10 or homeassistant.
+## default: localhost
+# hass_host = localhost
```
Comment (craigbarratt) [inline pyscript.conf:7]: While I do see you added defaults, I'd like for the three required parameter settings to not be commented.  Also, the comments should have single "#", not double.  It's ok for the hass_proxy setting to remain commented.

**Comment by craigbarratt** [comment]
Just checking if you saw my comments inline.

**Comment by fleXible** [comment]
Yes, I saw them. Just haven't had time to do the changes.

**Pull request merged**